core.md

```diff
@@ -65,7 +65,7 @@ A member may choose to become inactive and appoint another in their place until
 
 At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has fresh insight from a new team member with a different view on situations, and thus does not stagnate over time.
 
-Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the ranked voting system. It is intended that at least one, and at most three members will be promoted at a time.
+Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the [Helios voting system](https://heliosvoting.org/). It is intended that at least one, and at most three members will be promoted at a time.
```

Comment:
Could we perhaps amend the start of this sentence to make it clear that we're basing off our voting process on PEP 8104 (without having to hover over the link to figure that out)? Something like `Once a year, an internal voting process (based off of [PEP 8104](...)) will...`

Reply:
We should probably refer to the wording in [PEP13](https://peps.python.org/pep-0013/) for our own governance.